settings/storage.py

```diff
@@ -0,0 +1,83 @@
+import json
+import sqlite3
+from datetime import datetime
+
+from settings.detection import DetectionSettings
+from utils.logging import main_logger
+
+logger = main_logger.getChild("settings.storage")
+
+
+class SettingsStorage:
+    db_path: str
+
+    def __init__(self, db_path: str):
+        self.db_path = db_path
+        storage_conn = sqlite3.connect(self.db_path)
+        self._setup_storage(storage_conn)
+
+    def _setup_storage(self, storage_conn):
+        sql_q_db = """
+            CREATE TABLE IF NOT EXISTS "settings" (
+                "uuid"	string,
+                "settings" string,
+                "timestamp" string
+            );"""
+
+        storage_conn.execute(sql_q_db)
+        storage_conn.commit()
```

Comment:
Doesnt it autocommit?

Reply:
No, actually. I forgot to add it and was confused why nothing was saving.